src/crate/crash/command.py

```diff
@@ -481,11 +481,22 @@ def main():
     sys.exit(cmd.exit_code)
 
 def _create_cmd(crate_hosts, error_trace, output_writer, is_tty, args):
+
+    # Validate paths for certificate and key files
+    cert_file = args.cert_file
```

Comment:
Did you consider using the `type` functionality of argparse?
E.g. something like this:

```
def file_with_permissions(f):
    if not os.access(f, os.R_OK):
        raise Exception("foo")
    return f


parser = argparse.ArgumentParser()
parser.add_argument('--file', type=file_with_permissions, required=True)
try:
    parser.parse_args()
except Exception as e:
    print(str(e))
```